stable/dynamic-gateway-service/templates/statefulset.yaml

```diff
@@ -121,12 +128,19 @@ spec:
               find /drouter/ramdisk2/mnt -type d -exec chown 101:101 {} \;
 {{- end }}
               find /drouter/ramdisk2/mnt -type d -exec chmod 0777 {} \;
+{{- end }}
+{{- else }}
+              echo "ERROR: USER MUST SPECIFY A DATAPOWER MONITOR IMAGE"
```

Comment:
Just a note: this error will not be displayed by `kubectl logs <pod-name>` as the initcontainer is not the default logging entity. The logs will need to be obtained via `kubectl logs <pod-name> -c dpinit`, which won't be immediately apparent to most users.